scripts/env/dev/init.sh

```diff
@@ -186,7 +191,14 @@ cp -f ./hosts ./hosts_local
 sed -i "s/^xx.xx.xx.xx/127.0.0.1/" ./hosts_local
 
 # Setup dev env
-ansible-playbook -i ./hosts_local --connection local setup_s3dev_centos77_8.yml -v  -k --extra-vars "s3_src=${S3_SRC_DIR}"
+if [ $ansible_automation -eq 1 ]
+then
+   OPENLDAP_PASSWD=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 6 | head -n 1)
+   LDAPADMIN_PASSWD=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 7 | head -n 1)
+   ansible-playbook -i ./hosts_local --connection local setup_s3dev_centos77_8.yml -v --extra-vars "s3_src=${S3_SRC_DIR} openldappasswd=$OPENLDAP_PASSWD ldapiamadminpasswd=$LDAPADMIN_PASSWD"
```

Comment:
what about centos7.8, wil lthis work on centos7.8 as well ? I am talking about setup_s3dev_centos77_8.yml file.

Reply:
yes it will work it is tested on 7.8...u mean this yml file is different for 7.8?

Reply:
If its tested then fine. I don't know about this file either. I just commented in general.

You can resolve this conversation if no action needed.